src/containers/home/Home.test.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable testing-library/prefer-screen-queries */
+/* eslint-disable testing-library/render-result-naming-convention */
+
+import { render, fireEvent, screen } from "@testing-library/react";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+
+import Home from "./Home";
+import store from "../../store/store";
+
+import { AuthContext } from "../auth/AuthProvider";
+import { checkProps, getByTestId } from "../../utils/TestUtils";
+import * as userSelector from "../../store/selectores/user";
+
+const defaultProps = {};
+
+const defaultContextValues = {
+  isProcessing: false,
+  hasPermission: jest.fn(() => true),
+  setAuth: jest.fn(),
+  setIsBackendAuthorized: jest.fn(),
+  signOut: jest.fn(),
+};
+
+const setup = (props = {}, contextValues = {}) => {
+  const setupProps = { ...defaultProps, ...props };
+  const setupContextValues = { ...defaultContextValues, ...contextValues };
+
+  // Mock the userName selector here
+  userSelector.userName = jest.fn(() => "TestUserName");
```

Comment:
To simulate redux state in testing look at src/components/common/toastAlerts/ToastAlerts.test.js in empyreal project

Reply:
Fixed